test-network/prometheus-grafana/grafana/provisioning/dashboards/hlf-performances.json

```diff
@@ -0,0 +1,2407 @@
+		},
+		"lines": true,
+		"linewidth": 1,
+		"links": [],
+		"nullPointMode": "null as zero",
+		"options": {
+		  "alertThreshold": true
+		},
+		"percentage": false,
+		"pluginVersion": "8.2.2",
+		"pointradius": 5,
+		"points": false,
+		"renderer": "flot",
+		"seriesOverrides": [],
+		"spaceLength": 10,
+		"stack": true,
+		"steppedLine": false,
+		"targets": [
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(container_cpu_system_seconds_total[1m]))/8",
+			"hide": true,
+			"interval": "",
+			"intervalFactor": 2,
+			"legendFormat": "a",
+			"refId": "B",
+			"step": 120
+		  },
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(container_cpu_system_seconds_total{name=~\".+\"}[1m]))/8",
+			"hide": true,
+			"interval": "",
+			"intervalFactor": 2,
+			"legendFormat": "nur container",
+			"refId": "F",
+			"step": 10
+		  },
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(container_cpu_system_seconds_total{id=\"/\"}[1m]))/8",
+			"hide": true,
+			"interval": "",
+			"intervalFactor": 2,
+			"legendFormat": "nur docker host",
+			"metric": "",
+			"refId": "A",
+			"step": 20
+		  },
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(process_cpu_seconds_total[$interval]))/8 * 100",
+			"hide": true,
+			"interval": "",
+			"intervalFactor": 2,
+			"legendFormat": "host",
+			"metric": "",
+			"refId": "C",
+			"step": 600
+		  },
+		  {
+			"expr": "sum(rate(container_cpu_system_seconds_total{name=~\".+\"}[1m])) + sum(rate(container_cpu_system_seconds_total{id=\"/\"}[1m])) + sum(rate(process_cpu_seconds_total[1m]))",
+			"hide": true,
+			"intervalFactor": 2,
+			"legendFormat": "",
+			"refId": "D",
+			"step": 120
+		  },
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(container_cpu_usage_seconds_total{name=~\".+\"}[$interval]))/8 * 100",
+			"hide": false,
+			"interval": "",
+			"legendFormat": "",
+			"refId": "E"
+		  }
+		],
+		"thresholds": [],
+		"timeFrom": null,
+		"timeRegions": [],
+		"timeShift": null,
+		"title": "CPU Usage",
+		"tooltip": {
+		  "msResolution": true,
+		  "shared": true,
+		  "sort": 0,
+		  "value_type": "cumulative"
+		},
+		"type": "graph",
+		"xaxis": {
+		  "buckets": null,
+		  "mode": "time",
+		  "name": null,
+		  "show": false,
+		  "values": []
+		},
+		"yaxes": [
+		  {
+			"$$hashKey": "object:7341",
+			"format": "percent",
+			"label": "",
+			"logBase": 1,
+			"max": null,
+			"min": null,
+			"show": true
+		  },
+		  {
+			"$$hashKey": "object:7342",
+			"format": "short",
+			"label": null,
+			"logBase": 1,
+			"max": null,
+			"min": null,
+			"show": false
+		  }
+		],
+		"yaxis": {
+		  "align": false,
+		  "alignLevel": null
+		}
+	  },
+	  {
+		"aliasColors": {},
+		"bars": false,
+		"dashLength": 10,
+		"dashes": false,
+		"datasource": "Prometheus",
+		"editable": true,
+		"error": false,
+		"fieldConfig": {
+		  "defaults": {},
+		  "overrides": [
+			{
+			  "matcher": {
+				"id": "byRegexp",
+				"options": "dev-peer0.org1.example.com-fixed-asset-.*"
```

Comment:
Shouldn't this be `dev-peer0.org1.example.com-*` There is not instructions to deploy the fixed-asset chaincode, and they should be able to deploy any CC the wish.